protocols/gossipsub/src/config.rs

```diff
@@ -0,0 +1,251 @@
+// Copyright 2018 Parity Technologies (UK) Ltd.
```

Comment:
(feel free to use a different copyright)

Reply:
Modified. Thanks